public/js/tracking.js

```diff
@@ -0,0 +1,90 @@
+'use strict';
+
+/* global ga */
+
+var settings = require('settings');
+let instance = null;
+
+class Tracking {
```

Comment:
spiffy! Doesn't this file need to end in ``.jsx`` though? It seemed like the bable loader will not load js files but maybe I missed it.